server/tournaments/index.js

```diff
@@ -83,7 +84,8 @@ class Tournament extends Rooms.RoomGame {
 		this.playerCap = (playerCap ? parseInt(playerCap) : Config.tourdefaultplayercap) || 0;
 
 		/** @type {string} */
-		this.format = formatId;
+		// This will sometimes be sent alone in updates as "format", if the tour doesn't have a custom name
+		this.name = name || formatId;
 		this.originalFormat = formatId;
```

Comment:
Apparently there's already a `format` and `originalFormat`? How did they differ in the past?

Reply:
As far as I know, `format` will be the format unless it isn't, in which case `originalFormat` will get sent in updates as `teambuilderFormat` (not to be confused with Tournament#teambuilderFormat, which is used to store custom rules)  so the client knows what teambuilder to use.

Reply:
Oh, I see, `originalFormat` doesn't include custom rules. That's the only difference, right?

Reply:
(If so, can you add doc comments to that effect, so I don't get confused again in the future?)

Reply:
It looks like `teambuilderFormat` is `"gen7ou@@@!speciesclause"`, `originalFormat` is `"gen7ou"`. I _still_ don't know what `format` is for.